faf-commons-data/src/main/java/com/faforever/commons/replay/ReplayMetadata.java

```diff
@@ -0,0 +1,67 @@
+package com.faforever.commons.replay;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class is meant to be serialized/deserialized from/to JSON.
+ */
+@Data
+public class ReplayMetadata {
+  private CompressionType compression;
+  private String host;
+  private Integer uid;
+  private String title;
+  private String mapname;
+  private GameStatus state;
+  //TODO what is this?
+  private Boolean[] options;
+  // FAF calls this "game_type" but it's actually the victory condition.
+  private VictoryCondition gameType;
```

Comment:
Let's rename it then and use a @JsonProperty here as well

Reply:
Done